website/static/css/site.css

```diff
@@ -2778,3 +2778,21 @@ span#profileFullname{
         position: relative;
     }
 }
+
+
+/* Settings side-nav positions */
+#profileSettings .affix {
+    top: 80px;
+}
+
+#accountSettings .affix {
```

Comment:
Using IDs in CSS rules is a code smell. Use CSS classes for re-usability.

Something like

```css
.affix.osf-affix {
    top: 80px
}
```